src/database/admin.rs

```diff
@@ -360,10 +361,21 @@ impl AdminDatabase {
             ":pubkey": node.pubkey,
             ":status": node.status
         })?;
-
+        dbg!("Wrote node to the database");
+        
         Ok(self.connection.last_insert_rowid())
     }
 
+    pub fn clear_pending_node(&mut self) -> Result<(), Error> {
```

Comment:
So this is an error state I could pretty reliably put my system into.

Basically it goes

- Create node in the sensei sqlite DB (empty string pubkey)
- Create the actual lightning node (gets a pubkey)
- Update the sensei sqlite record with the pubkey

If there was ever a problem creating the node in the lightning node or updating the sqlite record, then there is an existing empty-string-for-a-pubkey "node" that's basically WIP for the system, but a new node can never be created because it can't write any more "new node with empty string pubkey".  So I made this to be able to "clean up after itself."

I actually think that's probably a wrong approach, and we should have a different table for like, "nodes to create" and then a "nodes created" and it's not just, "update the node record in place with a pubkey", but that's a bit larger scope.